codegen-core/common-test-models/naming-obstacle-course-prelude-structs.smithy

```diff
@@ -0,0 +1,125 @@
+$version: "1.0"
+namespace crate
+
+use smithy.test#httpRequestTests
+use smithy.test#httpResponseTests
+use aws.protocols#awsJson1_1
+use aws.api#service
+use smithy.framework#ValidationException
+
+/// Confounds model generation machinery by using structs named after every item in the Rust prelude
+@awsJson1_1
+@service(sdkId: "Config")
+service Config {
```

Comment:
It feels like there's not a need for having these be two separate integration tests.